tutor/src/screens/student-dashboard/event-row.jsx

```diff
@@ -45,22 +45,20 @@ EventTime.propTypes = {
 };
 
 export default
+@withRouter
```

Comment:
Does `withRouter` set `props.history` automatically?

Reply:
yes, it provides the `history` and `location` props.  https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md

The updated react-router uses new-style React.Context https://reactjs.org/docs/context.html